### ISSUE TITLE
Error fetching token

### ISSUE THREAD

**Issue by andrewroth** [docusign/eg-01-ruby-jwt#3]
Hi,

I'm trying to run the examples.  I've set up ds_config.rb as per instructions.  When I run it I get:

Sending an envelope...
Starting up: fetching token
DocuSign SDK Error!
   code: 400
   message: {"error":"consent_required"}

I was expecting at some point that it out output a URL that I would have to go to to authenticate and give permission to the integration key/app I added, but there seems to such point.  Can you provide any suggestions or help?

**Comment by LarryKlugerDS** [comment]
Hi, The ruby example doesn't yet output the url for manual consent. Here's the url you'd use with the sandbox:

````
consent_url = `https://account-d.docusign.com/oauth/auth?response_type=code&` +
              `scope=${consent_scopes}&client_id=${dsConfig.client_id}&` +
              `redirect_uri=${dsConfig.oauth_consent_redirect_URI}`;
````

Where consent_scopes is `signature%20impersonation`, client_id is your integration key, and redirect_uri is a redirect uri you set with the integration key via the admin tool. (I suggest https://www.docusign.com for this purpose.)

For production, it is better to grant blanket consent by using the organization administration tool.

**Comment by andrewroth** [comment]
Thanks for the quick reply.  I am trying this out on the sandbox.  I believe I already authorized this particular key before as I went through the login and approval for another gem.  When I go to that url now, it just sends me to the redirect uri with a token, so that seems to work.  I tried main.rb again and got the same error, though.  Is there more steps that need to be done?  Should I paste the token it gives back in the url somewhere?
